style.css

```
@@ -1,434 +1,580 @@
+    font-size: 3rem;
+    margin-bottom: 1rem;
+    animation-delay: 0.2s;
+}
+
+.hero-section p {
+    animation-delay: 0.4s;
+}
+
+.hero-section .cta-button {
+    animation-delay: 0.6s;
+    animation: fadeIn 0.8s ease forwards, pulse 2s infinite ease-in-out;
```

Biome (1.9.4)

[error] Unexpected shorthand property animation after animation-delay



(lint/suspicious/noShorthandPropertyOverrides)